src/emulator/functionsEmulator.ts

```diff
@@ -698,6 +689,19 @@ You can probably fix this by running "npm install ${
     return this.triggers;
   }
 
+  getBaseBundle(): FunctionsRuntimeBundle {
```

Comment:
This change allows us to continually check for the port information.  So during `start()` this is the same as before but now during `connect()` we know which other emulators are running and send that info to the runtime properly.

Reply:
neat